editorconfig-maven-plugin/src/main/java/org/ec4j/maven/AbstractEditorconfigMojo.java

```diff
@@ -323,7 +323,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                 }
             }
         }
-        return result.toArray(new String[result.size()]);
+        return result.toArray(new String[0]);
```

Comment:
Note that this is supposedly faster. See, e.g., https://stackoverflow.com/questions/22913122/list-toarrayobject-performance.